packages/iTwinUI-react/src/types/react-table-config.ts

```diff
@@ -86,9 +86,9 @@ declare module 'react-table' {
       UseRowSelectOptions<D>,
       UseExpandedOptions<D>,
       UseFiltersOptions<D>,
-      // UseGlobalFiltersOptions<D>,
       // UseGroupByOptions<D>,
       UsePaginationOptions<D>,
+      UseGlobalFiltersOptions<D>,
```

Comment:
`UseGlobalFiltersColumnOptions` would also be useful for disabling global filter for individual columns.